dymos/examples/racecar/test/test_racecar.py

```diff
@@ -40,319 +40,155 @@ def test_racecar_for_docs(self):
-        plot_track_with_data(delta, s)
-
-        # Plot the main vehicle telemetry
-        fig, axes = plt.subplots(nrows=4, ncols=1, figsize=(15, 8))
-
-        # Velocity vs s
-        axes[0].plot(s,
-                     p.get_val('traj.phase0.timeseries.V'), label='solution')
-
-        axes[0].set_xlabel('s (m)')
-        axes[0].set_ylabel('V (m/s)')
-        axes[0].grid()
-        axes[0].set_xlim(0, s_final)
-
-        # n vs s
-        axes[1].plot(s,
-                     p.get_val('traj.phase0.timeseries.n', units='m'), label='solution')
-
-        axes[1].set_xlabel('s (m)')
-        axes[1].set_ylabel('n (m)')
-        axes[1].grid()
-        axes[1].set_xlim(0, s_final)
-
-        # throttle vs s
-        axes[2].plot(s, thrust)
-
-        axes[2].set_xlabel('s (m)')
-        axes[2].set_ylabel('thrust')
-        axes[2].grid()
-        axes[2].set_xlim(0, s_final)
-
-        # delta vs s
-        axes[3].plot(s,
-                     p.get_val('traj.phase0.timeseries.delta', units=None),
-                     label='solution')
-
-        axes[3].set_xlabel('s (m)')
-        axes[3].set_ylabel('delta')
-        axes[3].grid()
-        axes[3].set_xlim(0, s_final)
-
-        plt.tight_layout()
-
-        # Performance constraint plot. Tire friction and power constraints
-        fig, axes = plt.subplots(nrows=1, ncols=1, figsize=(15, 4))
-        plt.subplots_adjust(right=0.82, bottom=0.14, top=0.97, left=0.07)
-
-        axes.plot(s,
-                  p.get_val('traj.phase0.timeseries.c_fl', units=None), label='c_fl')
-        axes.plot(s,
-                  p.get_val('traj.phase0.timeseries.c_fr', units=None), label='c_fr')
-        axes.plot(s,
-                  p.get_val('traj.phase0.timeseries.c_rl', units=None), label='c_rl')
-        axes.plot(s,
-                  p.get_val('traj.phase0.timeseries.c_rr', units=None), label='c_rr')
-
-        axes.plot(s, power / pmax, label='Power')
-
-        axes.legend(bbox_to_anchor=(1.04, 0.5), loc="center left")
-        axes.set_xlabel('s (m)')
-        axes.set_ylabel('Performance constraints')
-        axes.grid()
-        axes.set_xlim(0, s_final)
+        txs = {'radau': dm.Radau(num_segments=50, order=3, compressed=True),
+               'gauss-lobatto': dm.GaussLobatto(num_segments=50, order=3, compressed=True),
+               'birkhoff': dm.Birkhoff(grid=dm.BirkhoffGrid(num_nodes=50))}
+
+        for tx_name, tx in txs.items():
+            with self.subTest(tx_name):
+
+                # Define the OpenMDAO problem
+                p = om.Problem(model=om.Group())
+
+                # Define a Trajectory object
+                traj = dm.Trajectory()
+                p.model.add_subsystem('traj', subsys=traj)
+
+                # Define a Dymos Phase object with GaussLobatto Transcription
+                # tx = dm.Radau(num_segments=50, order=3, compressed=True)
+                tx = dm.Birkhoff(grid=dm.BirkhoffGrid(num_nodes=50))
```

Comment:
is this some leftover debugging?  it replaces the `tx` parameter for the subTest?

Reply:
ah yep